pages/linux/qm-cleanup.md

```diff
@@ -0,0 +1,8 @@
+# qm cleanup
+
+> Cleans up resources like tap devices, vgpus, etc. Called after a vm shuts down, crashes, etc.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Clean up resources like tap devices, vgpus, etc. after vm shuts down, crashes, etc:
```

Comment:
```suggestion
- Clean up resources:
```
Since we are already mentioning it in the page description, I don't think we need to add it again to the command description.